build/linux/installer/conf/telegraf.conf

```diff
@@ -465,7 +465,7 @@
   ## By default, telegraf will gather stats for all devices including
   ## disk partitions.
   ## Setting devices will restrict the stats to the specified devices.
-  devices = ["sd[a-z][0-9]"]
+  devices = ["sd[a-z][0-9]", "xvda"]
```

Comment:
we should add regex to cover all different devices of xvd. Refer to this https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html

Reply:
added and changed to xvd[a-z]